assets/elasticsearch/measure-compressed-size.py

```
@@ -0,0 +1,9 @@
+import time
+
+import requests
+
+collection_name = "elasticsearch_clp_bench"
+
+time.sleep(5)
+response = requests.get(f"http://localhost:9202/{collection_name}/_stats").json()
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)